tests/components/plex/test_config_flow.py

```diff
@@ -221,23 +239,34 @@ def init_config_flow(hass):
 async def test_no_servers_found(hass):
     """Test when no servers are on an account."""
 
+    await async_setup_component(hass, "http", {"http": {}})
+
     result = await hass.config_entries.flow.async_init(
         config_flow.DOMAIN, context={"source": "user"}
     )
-
     assert result["type"] == "form"
     assert result["step_id"] == "user"
 
     mm_plex_account = MagicMock()
     mm_plex_account.resources = Mock(return_value=[])
 
-    with patch("plexapi.myplex.MyPlexAccount", return_value=mm_plex_account):
+    with patch(
+        "plexapi.myplex.MyPlexAccount", return_value=mm_plex_account
+    ), asynctest.patch("plexauth.PlexAuth.initiate_auth"), asynctest.patch(
+        "plexauth.PlexAuth.token", return_value=MOCK_TOKEN
+    ):
 
         result = await hass.config_entries.flow.async_configure(
-            result["flow_id"],
-            user_input={CONF_TOKEN: MOCK_TOKEN, "manual_setup": False},
+            result["flow_id"], user_input={"manual_setup": False}
         )
+        assert result["type"] == "external"
 
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"], user_input={"manual_setup": False}
```

Comment:
There's no user input when continuing after the external step.

Reply:
Thanks. Probably a lazy copy/paste that still worked.